### PR TITLE
Vye 374 move code to profile

### PR DESCRIPTION
**Pull request by wafimohamed**
**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be completed as written. Provide known gaps to testing that may raise the risk of merging to production.

## Summary

- _(Summarize the changes that have been made to the platform)_
 Moved Direct Deposit and Address code to the profile. It was requested by Mikki Northuis and the VA profile team to move the VYE address and direct deposit forms from the VYE application and into the VA profile section. This way all address/direct deposit forms are consolidated in one area. 
Based on this conversation, this PR was created to move the code for the related forms, so that the forms can be built into the VA profile. 
- _(If bug, how to reproduce)_
- _(What is the solution, why is this the solution)_
- _(Which team do you work for, does your team own the maintenance of this component?)_
- _(If using a flipper, what is the end date of the flipper being required/success criteria being targeted)_

## Related issue(s)
https://jira.devops.va.gov/browse/VYE-374
As:

Development Lead

I need:

To meet with Adam W from VA Profile to discuss duplicating our code in VA Profile  

So that:

We can maintain code ownership and complete a portion of the actions necessary to comply with VA Profile requirements to merge VYE Profile into VA Profile. This requires VYE to put the interface portion of VYE Benefits Profile onto VA Profile.

Acceptance Criteria:

Enter the acceptance criteria that will be used to test the user story.

AC 1: Complete a meeting with Adam W from VA Profile about duplicating our code in VA Profile so we maintain code ownership 
AC 2: Determine where VYE needs to move our code so it can be shared with VA Profile. Once determine, the location needs to be noted in the comments of this ticket. 
Testing Considerations:

There are no testing considerations 
Other considerations:

This is necessary to make the code accessible to VA Profile. The code must be accessible to VA Profile so the interface portion of VYE Benefits Profile can be moved over to comply with Platform. 

## Testing done

- _Describe what the old behavior was prior to the change_
- _Describe the steps required to verify your changes are working as expected_
- _Describe the tests completed and the results_
- _Exclusively stating 'Specs and automated tests passing' is NOT acceptable as appropriate testing
- _Optionally, provide a link to your [test plan](https://depo-platform-documentation.scrollhelp.site/developer-docs/create-a-test-plan-in-testrail) and [test execution records](https://depo-platform-documentation.scrollhelp.site/developer-docs/execute-tests-in-testrail)_

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| Mobile  |        |       |
| Desktop |        |       |

## What areas of the site does it impact?

*(Describe what parts of the site are impacted **if** code touched other areas)*

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

**Review by jsimonVA — DISMISSED** [review]
This code move is to create the VYE forms in the VA profile as discussed with profile team. LGTM

**Comment by adamwhitlock1** [comment]
Can you please fill out the main sections of the template with ticket that this is related to and a brief explanation of the aim of this PR? That way we have some documentation into this PR for future engineers if they need to look back at it.

Are there unit tests for these components/code that are also going to be moved over? It's cool if those come over with an additional PR, but just want to make sure that code coverage is maintained overall throughout the application.

File: src/applications/personalization/profile/components/direct-deposit/vye/reducers/personalInfo.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        isLoading: true,
+        error: null,
+      };
+    case FETCH_PERSONAL_INFO_SUCCESS:
+      return {
+        ...state,
+        personalInfo: action.response,
+        isLoading: false,
+      };
+    case UPDATE_PENDING_VERIFICATIONS:
```
Comment (adamwhitlock1) [inline src/applications/personalization/profile/components/direct-deposit/vye/reducers/personalInfo.js:36]: This action is flagged as not found in the actions file. Is this case still needed?

File: src/applications/personalization/profile/components/direct-deposit/vye/reducers/personalInfo.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        personalInfo: {
+          ...(state.personalInfo || {}),
+          'vye::UserInfo': {
+            ...((state.personalInfo && state.personalInfo['vye::UserInfo']) ||
+              defaultUserInfo['vye::UserInfo']),
+            pendingVerifications: action.payload,
+          },
+        },
+      };
+    case UPDATE_VERIFICATIONS:
```
Comment (adamwhitlock1) [inline src/applications/personalization/profile/components/direct-deposit/vye/reducers/personalInfo.js:48]: Same as above comment, is this case needed?

File: src/applications/personalization/profile/components/direct-deposit/vye/reducers/personalInfo.js (hunk 1 above)
Comment (wafimohamed) [reply src/applications/personalization/profile/components/direct-deposit/vye/reducers/personalInfo.js:36]: unit testing will be created once we show the address and the direct deposit in the UI in the profile page.

**Review by jsimonVA — DISMISSED** [review]

**Review by jsimonVA — APPROVED** [review]

**Review by adamwhitlock1 — APPROVED** [review]
Locally the Profile runs as expected on this branch and all tests pass.

I don't see anything that would block this PR. Approved, thanks @wafimohamed

**Pull request merged**